doc/en/how-to/usage.rst

```diff
@@ -200,28 +200,10 @@ hook was invoked:
 .. code-block:: pytest
 
     $ python myinvoke.py
-    .FEsxX.                                                              [100%]*** test run reporting finishing
-
-    ================================== ERRORS ==================================
-    _______________________ ERROR at setup of test_error _______________________
-
-        @pytest.fixture
-        def error_fixture():
-    >       assert 0
-    E       assert 0
-
-    test_example.py:6: AssertionError
-    ================================= FAILURES =================================
-    ________________________________ test_fail _________________________________
-
-        def test_fail():
-    >       assert 0
-    E       assert 0
-
-    test_example.py:14: AssertionError
-    ========================= short test summary info ==========================
-    FAILED test_example.py::test_fail - assert 0
-    ERROR test_example.py::test_error - assert 0
+    Traceback (most recent call last):
```

Comment:
Oh we missed a `import sys` in the code above it (I can't suggest a change because it is not part of the diff :/)